open-build-service-api/src/lib.rs

```diff
@@ -87,7 +87,12 @@ impl PackageCode {
     pub fn is_final(&self) -> bool {
         matches!(
             self,
-            Self::Disabled | Self::Succeeded | Self::Failed | Self::Excluded
+            Self::Broken
+                | Self::Disabled
+                | Self::Excluded
+                | Self::Failed
+                | Self::Succeeded
+                | Self::Unresolvable
```

Comment:
Unresolvable isn't final; that can change due to other builds

Reply:
This was taken from ci-package-builder's "final" statuses, I guess in this case I should remove it here and test for it explicitly in the runner?

Reply:
Yeah drop it here; that's a policy decision for the gitlab runner on how to handle it (mark failed? block? block iff there are builds running?)